app/scripts/modules/appengine/helpContents/appengineHelpContents.ts

```diff
@@ -32,6 +32,14 @@ module(APPENGINE_HELP_CONTENTS_REGISTRY, [
                 (Spinnaker load balancer) will be stopped. This option will be respected only if this server group will
                 be receiving all traffic and the previous server group is using manual scaling.`,
       },
+      {
+        key: 'appengine.loadBalancer.shardBy.cookie',
```

Comment:
Shard is the official term (https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services#ShardBy). The shardBy property of a traffic splitter determines which version a request is routed to, I think the request is still round-robined among the version's instances.